app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -277,4 +277,10 @@ def url_for_report(report)
   def timestamp
     Time.zone.now.strftime("%Y%m%d")
   end
+
+  def orders_and_fulfillment_default_filters
+    now = Time.zone.now
+    { completed_at_gt: (now - 1.month).beginning_of_day,
+      completed_at_lt: (now + 1.day).beginning_of_day }
+  end
```

Comment:
I'd be conservative and limit it to 1 week only. I haven't checked the numbers but a month can surely cause :boom: in many cases which would lead to a poor experience for the user.

Reply:
it's a requirement here:
https://github.com/openfoodfoundation/openfoodnetwork/issues/4315#issuecomment-535428200

@lin-d-hop would "1 week ago" be ok as default start date?

Reply:
ping @lin-d-hop @RachL

Reply:
there's also recent related discussion in the issue (#4315)